lms/djangoapps/grades/models.py

```diff
@@ -234,7 +234,7 @@ def __unicode__(self):
             self.user_id,
             self.course_version,
             self.usage_key,
-            self.visible_blocks.hashed,
+            self.visible_blocks.id,
```

Comment:
This didn't successfully lower the query count, did it?  I would think you need to do `self.visible_blocks_id` instead.